From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: PKI Metadata Updates Bot
 <mdb.chrome-pki-metadata-release-jobs@google.com>
Date: Sat, 3 Jun 2023 14:33:38 +0000
Subject: [PATCH] Automatic update from google3

Automatic update for 2023-06-03 UTC

Change-Id: Ib8d5d7a1bff3357391c09ffb99ee6df4b8273729
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4582494
Commit-Queue: PKI Metadata Updates Bot <mdb.chrome-pki-metadata-release-jobs@google.com>
Bot-Commit: PKI Metadata Updates Bot <mdb.chrome-pki-metadata-release-jobs@google.com>
Cr-Commit-Position: refs/heads/main@{#1152920}
---
 .../ssl/chrome_root_store/root_store.certs    | 391 ------------------
 net/data/ssl/chrome_root_store/root_store.md  |   6 +-
 .../chrome_root_store/root_store.textproto    |  26 +-
 3 files changed, 3 insertions(+), 420 deletions(-)

diff --git a/net/data/ssl/chrome_root_store/root_store.certs b/net/data/ssl/chrome_root_store/root_store.certs
index 55cb1ee0cad61..feacf5a71ed06 100644
--- a/net/data/ssl/chrome_root_store/root_store.certs
+++ b/net/data/ssl/chrome_root_store/root_store.certs
@@ -3116,134 +3116,6 @@ zMOl6W8KjptlwlCFtaOgUxLMVYdh84GuEEZhvUQhuMI9dM9+JDX6HAcOmz0iyu8x
 L4ysEr3vQCj8KWefshNPZiTEUxnpHikV7+ZtsH8tZ/3zbBt1RqPlShfppNcL
 -----END CERTIFICATE-----
 
-# b0bfd52bb0d7d9bd92bf5d4dc13da255c02c542f378365ea893911f55e55f23c
-Certificate:
-    Data:
-        Version: 3 (0x2)
-        Serial Number: 7667447206703254355 (0x6a683e9c519bcb53)
-        Signature Algorithm: sha256WithRSAEncryption
-        Issuer: C = TR, L = Ankara, O = E-Tu\C4\9Fra EBG Bili\C5\9Fim Teknolojileri ve Hizmetleri A.\C5\9E., OU = E-Tugra Sertifikasyon Merkezi, CN = E-Tugra Certification Authority
-        Validity
-            Not Before: Mar  5 12:09:48 2013 GMT
-            Not After : Mar  3 12:09:48 2023 GMT
-        Subject: C = TR, L = Ankara, O = E-Tu\C4\9Fra EBG Bili\C5\9Fim Teknolojileri ve Hizmetleri A.\C5\9E., OU = E-Tugra Sertifikasyon Merkezi, CN = E-Tugra Certification Authority
-        Subject Public Key Info:
-            Public Key Algorithm: rsaEncryption
-                RSA Public-Key: (4096 bit)
-                Modulus:
-                    00:e2:f5:3f:93:05:51:1e:85:62:54:5e:7a:0b:f5:
-                    18:07:83:ae:7e:af:7c:f7:d4:8a:6b:a5:63:43:39:
-                    b9:4b:f7:c3:c6:64:89:3d:94:2e:54:80:52:39:39:
-                    07:4b:4b:dd:85:07:76:87:cc:bf:2f:95:4c:cc:7d:
-                    a7:3d:bc:47:0f:98:70:f8:8c:85:1e:74:8e:92:6d:
-                    1b:40:d1:99:0d:bb:75:6e:c8:a9:6b:9a:c0:84:31:
-                    af:ca:43:cb:eb:2b:34:e8:8f:97:6b:01:9b:d5:0e:
-                    4a:08:aa:5b:92:74:85:43:d3:80:ae:a1:88:5b:ae:
-                    b3:ea:5e:cb:16:9a:77:44:c8:a1:f6:54:68:ce:de:
-                    8f:97:2b:ba:5b:40:02:0c:64:17:c0:b5:93:cd:e1:
-                    f1:13:66:ce:0c:79:ef:d1:91:28:ab:5f:a0:12:52:
-                    30:73:19:8e:8f:e1:8c:07:a2:c3:bb:4a:f0:ea:1f:
-                    15:a8:ee:25:cc:a4:46:f8:1b:22:ef:b3:0e:43:ba:
-                    2c:24:b8:c5:2c:5c:d4:1c:f8:5d:64:bd:c3:93:5e:
-                    28:a7:3f:27:f1:8e:1e:d3:2a:50:05:a3:55:d9:cb:
-                    e7:39:53:c0:98:9e:8c:54:62:8b:26:b0:f7:7d:8d:
-                    7c:e4:c6:9e:66:42:55:82:47:e7:b2:58:8d:66:f7:
-                    07:7c:2e:36:e6:50:1c:3f:db:43:24:c5:bf:86:47:
-                    79:b3:79:1c:f7:5a:f4:13:ec:6c:f8:3f:e2:59:1f:
-                    95:ee:42:3e:b9:ad:a8:32:85:49:97:46:fe:4b:31:
-                    8f:5a:cb:ad:74:47:1f:e9:91:b7:df:28:04:22:a0:
-                    d4:0f:5d:e2:79:4f:ea:6c:85:86:bd:a8:a6:ce:e4:
-                    fa:c3:e1:b3:ae:de:3c:51:ee:cb:13:7c:01:7f:84:
-                    0e:5d:51:94:9e:13:0c:b6:2e:a5:4c:f9:39:70:36:
-                    6f:96:ca:2e:0c:44:55:c5:ca:fa:5d:02:a3:df:d6:
-                    64:8c:5a:b3:01:0a:a9:b5:0a:47:17:ff:ef:91:40:
-                    2a:8e:a1:46:3a:31:98:e5:11:fc:cc:bb:49:56:8a:
-                    fc:b9:d0:61:9a:6f:65:6c:e6:c3:cb:3e:75:49:fe:
-                    8f:a7:e2:89:c5:67:d7:9d:46:13:4e:31:76:3b:24:
-                    b3:9e:11:65:86:ab:7f:ef:1d:d4:f8:bc:e7:ac:5a:
-                    5c:b7:5a:47:5c:55:ce:55:b4:22:71:5b:5b:0b:f0:
-                    cf:dc:a0:61:64:ea:a9:d7:68:0a:63:a7:e0:0d:3f:
-                    a0:af:d3:aa:d2:7e:ef:51:a0:e6:51:2b:55:92:15:
-                    17:53:cb:b7:66:0e:66:4c:f8:f9:75:4c:90:e7:12:
-                    70:c7:45
-                Exponent: 65537 (0x10001)
-        X509v3 extensions:
-            X509v3 Subject Key Identifier: 
-                2E:E3:DB:B2:49:D0:9C:54:79:5C:FA:27:2A:FE:CC:4E:D2:E8:4E:54
-            X509v3 Basic Constraints: critical
-                CA:TRUE
-            X509v3 Authority Key Identifier: 
-                keyid:2E:E3:DB:B2:49:D0:9C:54:79:5C:FA:27:2A:FE:CC:4E:D2:E8:4E:54
-
-            X509v3 Key Usage: critical
-                Certificate Sign, CRL Sign
-    Signature Algorithm: sha256WithRSAEncryption
-         05:37:3a:f4:4d:b7:45:e2:45:75:24:8f:b6:77:52:e8:1c:d8:
-         10:93:65:f3:f2:59:06:a4:3e:1e:29:ec:5d:d1:d0:ab:7c:e0:
-         0a:90:48:78:ed:4e:98:03:99:fe:28:60:91:1d:30:1d:b8:63:
-         7c:a8:e6:35:b5:fa:d3:61:76:e6:d6:07:4b:ca:69:9a:b2:84:
-         7a:77:93:45:17:15:9f:24:d0:98:13:12:ff:bb:a0:2e:fd:4e:
-         4c:87:f8:ce:5c:aa:98:1b:05:e0:00:46:4a:82:80:a5:33:8b:
-         28:dc:ed:38:d3:df:e5:3e:e9:fe:fb:59:dd:61:84:4f:d2:54:
-         96:13:61:13:3e:8f:80:69:be:93:47:b5:35:43:d2:5a:bb:3d:
-         5c:ef:b3:42:47:cd:3b:55:13:06:b0:09:db:fd:63:f6:3a:88:
-         0a:99:6f:7e:e1:ce:1b:53:6a:44:66:23:51:08:7b:bc:5b:52:
-         a2:fd:06:37:38:40:61:8f:4a:96:b8:90:37:f8:66:c7:78:90:
-         00:15:2e:8b:ad:51:35:53:07:a8:6b:68:ae:f9:4e:3c:07:26:
-         cd:08:05:70:cc:39:3f:76:bd:a5:d3:67:26:01:86:a6:53:d2:
-         60:3b:7c:43:7f:55:8a:bc:95:1a:c1:28:39:4c:1f:43:d2:91:
-         f4:72:59:8a:b9:56:fc:3f:b4:9d:da:70:9c:76:5a:8c:43:50:
-         ee:8e:30:72:4d:df:ff:49:f7:c6:a9:67:d9:6d:ac:02:11:e2:
-         3a:16:25:a7:58:08:cb:6f:53:41:9c:48:38:47:68:33:d1:d7:
-         c7:8f:d4:74:21:d4:c3:05:90:7a:ff:ce:96:88:b1:15:29:5d:
-         23:ab:d0:60:a1:12:4f:de:f4:17:cd:32:e5:c9:bf:c8:43:ad:
-         fd:2e:8e:f1:af:e2:f4:98:fa:12:1f:20:d8:c0:a7:0c:85:c5:
-         90:f4:3b:2d:96:26:b1:2c:be:4c:ab:eb:b1:d2:8a:c9:db:78:
-         13:0f:1e:09:9d:6d:8f:00:9f:02:da:c1:fa:1f:7a:7a:09:c4:
-         4a:e6:88:2a:97:9f:89:8b:fd:37:5f:5f:3a:ce:38:59:86:4b:
-         af:71:0b:b4:d8:f2:70:4f:9f:32:13:e3:b0:a7:57:e5:da:da:
-         43:cb:84:34:f2:28:c4:ea:6d:f4:2a:ef:c1:6b:76:da:fb:7e:
-         bb:85:3c:d2:53:c2:4d:be:71:e1:45:d1:fd:23:67:0d:13:75:
-         fb:cf:65:67:22:9d:ae:b0:09:d1:09:ff:1d:34:bf:fe:23:97:
-         37:d2:39:fa:3d:0d:06:0b:b4:db:3b:a3:ab:6f:5c:1d:b6:7e:
-         e8:b3:82:34:ed:06:5c:24
------BEGIN CERTIFICATE-----
-MIIGSzCCBDOgAwIBAgIIamg+nFGby1MwDQYJKoZIhvcNAQELBQAwgbIxCzAJBgNV
-BAYTAlRSMQ8wDQYDVQQHDAZBbmthcmExQDA+BgNVBAoMN0UtVHXEn3JhIEVCRyBC
-aWxpxZ9pbSBUZWtub2xvamlsZXJpIHZlIEhpem1ldGxlcmkgQS7Fni4xJjAkBgNV
-BAsMHUUtVHVncmEgU2VydGlmaWthc3lvbiBNZXJrZXppMSgwJgYDVQQDDB9FLVR1
-Z3JhIENlcnRpZmljYXRpb24gQXV0aG9yaXR5MB4XDTEzMDMwNTEyMDk0OFoXDTIz
-MDMwMzEyMDk0OFowgbIxCzAJBgNVBAYTAlRSMQ8wDQYDVQQHDAZBbmthcmExQDA+
-BgNVBAoMN0UtVHXEn3JhIEVCRyBCaWxpxZ9pbSBUZWtub2xvamlsZXJpIHZlIEhp
-em1ldGxlcmkgQS7Fni4xJjAkBgNVBAsMHUUtVHVncmEgU2VydGlmaWthc3lvbiBN
-ZXJrZXppMSgwJgYDVQQDDB9FLVR1Z3JhIENlcnRpZmljYXRpb24gQXV0aG9yaXR5
-MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEA4vU/kwVRHoViVF56C/UY
-B4Oufq9899SKa6VjQzm5S/fDxmSJPZQuVIBSOTkHS0vdhQd2h8y/L5VMzH2nPbxH
-D5hw+IyFHnSOkm0bQNGZDbt1bsipa5rAhDGvykPL6ys06I+XawGb1Q5KCKpbknSF
-Q9OArqGIW66z6l7LFpp3RMih9lRozt6Plyu6W0ACDGQXwLWTzeHxE2bODHnv0ZEo
-q1+gElIwcxmOj+GMB6LDu0rw6h8VqO4lzKRG+Bsi77MOQ7osJLjFLFzUHPhdZL3D
-k14opz8n8Y4e0ypQBaNV2cvnOVPAmJ6MVGKLJrD3fY185MaeZkJVgkfnsliNZvcH
-fC425lAcP9tDJMW/hkd5s3kc91r0E+xs+D/iWR+V7kI+ua2oMoVJl0b+SzGPWsut
-dEcf6ZG33ygEIqDUD13ieU/qbIWGvaimzuT6w+Gzrt48Ue7LE3wBf4QOXVGUnhMM
-ti6lTPk5cDZvlsouDERVxcr6XQKj39ZkjFqzAQqptQpHF//vkUAqjqFGOjGY5RH8
-zLtJVor8udBhmm9lbObDyz51Sf6Pp+KJxWfXnUYTTjF2OySznhFlhqt/7x3U+Lzn
-rFpct1pHXFXOVbQicVtbC/DP3KBhZOqp12gKY6fgDT+gr9Oq0n7vUaDmUStVkhUX
-U8u3Zg5mTPj5dUyQ5xJwx0UCAwEAAaNjMGEwHQYDVR0OBBYEFC7j27JJ0JxUeVz6
-Jyr+zE7S6E5UMA8GA1UdEwEB/wQFMAMBAf8wHwYDVR0jBBgwFoAULuPbsknQnFR5
-XPonKv7MTtLoTlQwDgYDVR0PAQH/BAQDAgEGMA0GCSqGSIb3DQEBCwUAA4ICAQAF
-Nzr0TbdF4kV1JI+2d1LoHNgQk2Xz8lkGpD4eKexd0dCrfOAKkEh47U6YA5n+KGCR
-HTAduGN8qOY1tfrTYXbm1gdLymmasoR6d5NFFxWfJNCYExL/u6Au/U5Mh/jOXKqY
-GwXgAEZKgoClM4so3O0409/lPun++1ndYYRP0lSWE2ETPo+Aab6TR7U1Q9Jauz1c
-77NCR807VRMGsAnb/WP2OogKmW9+4c4bU2pEZiNRCHu8W1Ki/QY3OEBhj0qWuJA3
-+GbHeJAAFS6LrVE1Uweoa2iu+U48BybNCAVwzDk/dr2l02cmAYamU9JgO3xDf1WK
-vJUawSg5TB9D0pH0clmKuVb8P7Sd2nCcdlqMQ1DujjByTd//SffGqWfZbawCEeI6
-FiWnWAjLb1NBnEg4R2gz0dfHj9R0IdTDBZB6/86WiLEVKV0jq9BgoRJP3vQXzTLl
-yb/IQ639Lo7xr+L0mPoSHyDYwKcMhcWQ9DstliaxLL5Mq+ux0orJ23gTDx4JnW2P
-AJ8C2sH6H3p6CcRK5ogql5+Ji/03X186zjhZhkuvcQu02PJwT58yE+Owp1fl2tpD
-y4Q08ijE6m30Ku/Ba3ba+367hTzSU8JNvnHhRdH9I2cNE3X7z2VnIp2usAnRCf8d
-NL/+I5c30jn6PQ0GC7TbO6Orb1wdtn7os4I07QZcJA==
------END CERTIFICATE-----
-
 # 125609aa301da0a249b97a8239cb6a34216f44dcac9f3954b14292f2e8c8608f
 Certificate:
     Data:
@@ -5600,82 +5472,6 @@ LJstxabArahH9CdMOA0uG0k7UvToiIMrVCjU8jVStDKDYmlkDJGcn5fqdBb9HxEG
 mpv0
 -----END CERTIFICATE-----
 
-# f9e67d336c51002ac054c632022d66dda2e7e3fff10ad061ed31d8bbb410cfb2
-Certificate:
-    Data:
-        Version: 3 (0x2)
-        Serial Number: 1000 (0x3e8)
-        Signature Algorithm: sha1WithRSAEncryption
-        Issuer: C = HK, O = Hongkong Post, CN = Hongkong Post Root CA 1
-        Validity
-            Not Before: May 15 05:13:14 2003 GMT
-            Not After : May 15 04:52:29 2023 GMT
-        Subject: C = HK, O = Hongkong Post, CN = Hongkong Post Root CA 1
-        Subject Public Key Info:
-            Public Key Algorithm: rsaEncryption
-                RSA Public-Key: (2048 bit)
-                Modulus:
-                    00:ac:ff:38:b6:e9:66:02:49:e3:a2:b4:e1:90:f9:
-                    40:8f:79:f9:e2:bd:79:fe:02:bd:ee:24:92:1d:22:
-                    f6:da:85:72:69:fe:d7:3f:09:d4:dd:91:b5:02:9c:
-                    d0:8d:5a:e1:55:c3:50:86:b9:29:26:c2:e3:d9:a0:
-                    f1:69:03:28:20:80:45:22:2d:56:a7:3b:54:95:56:
-                    22:59:1f:28:df:1f:20:3d:6d:a2:36:be:23:a0:b1:
-                    6e:b5:b1:27:3f:39:53:09:ea:ab:6a:e8:74:b2:c2:
-                    65:5c:8e:bf:7c:c3:78:84:cd:9e:16:fc:f5:2e:4f:
-                    20:2a:08:9f:77:f3:c5:1e:c4:9a:52:66:1e:48:5e:
-                    e3:10:06:8f:22:98:e1:65:8e:1b:5d:23:66:3b:b8:
-                    a5:32:51:c8:86:aa:a1:a9:9e:7f:76:94:c2:a6:6c:
-                    b7:41:f0:d5:c8:06:38:e6:d4:0c:e2:f3:3b:4c:6d:
-                    50:8c:c4:83:27:c1:13:84:59:3d:9e:75:74:b6:d8:
-                    02:5e:3a:90:7a:c0:42:36:72:ec:6a:4d:dc:ef:c4:
-                    00:df:13:18:57:5f:26:78:c8:d6:0a:79:77:bf:f7:
-                    af:b7:76:b9:a5:0b:84:17:5d:10:ea:6f:e1:ab:95:
-                    11:5f:6d:3c:a3:5c:4d:83:5b:f2:b3:19:8a:80:8b:
-                    0b:87
-                Exponent: 65537 (0x10001)
-        X509v3 extensions:
-            X509v3 Basic Constraints: critical
-                CA:TRUE, pathlen:3
-            X509v3 Key Usage: critical
-                Digital Signature, Non Repudiation, Certificate Sign, CRL Sign
-    Signature Algorithm: sha1WithRSAEncryption
-         0e:46:d5:3c:ae:e2:87:d9:5e:81:8b:02:98:41:08:8c:4c:bc:
-         da:db:ee:27:1b:82:e7:6a:45:ec:16:8b:4f:85:a0:f3:b2:70:
-         bd:5a:96:ba:ca:6e:6d:ee:46:8b:6e:e7:2a:2e:96:b3:19:33:
-         eb:b4:9f:a8:b2:37:ee:98:a8:97:b6:2e:b6:67:27:d4:a6:49:
-         fd:1c:93:65:76:9e:42:2f:dc:22:6c:9a:4f:f2:5a:15:39:b1:
-         71:d7:2b:51:e8:6d:1c:98:c0:d9:2a:f4:a1:82:7b:d5:c9:41:
-         a2:23:01:74:38:55:8b:0f:b9:2e:67:a2:20:04:37:da:9c:0b:
-         d3:17:21:e0:8f:97:79:34:6f:84:48:02:20:33:1b:e6:34:44:
-         9f:91:70:f4:80:5e:84:43:c2:29:d2:6c:12:14:e4:61:8d:ac:
-         10:90:9e:84:50:bb:f0:96:6f:45:9f:8a:f3:ca:6c:4f:fa:11:
-         3a:15:15:46:c3:cd:1f:83:5b:2d:41:12:ed:50:67:41:13:3d:
-         21:ab:94:8a:aa:4e:7c:c1:b1:fb:a7:d6:b5:27:2f:97:ab:6e:
-         e0:1d:e2:d1:1c:2c:1f:44:e2:fc:be:91:a1:9c:fb:d6:29:53:
-         73:86:9f:53:d8:43:0e:5d:d6:63:82:71:1d:80:74:ca:f6:e2:
-         02:6b:d9:5a
------BEGIN CERTIFICATE-----
-MIIDMDCCAhigAwIBAgICA+gwDQYJKoZIhvcNAQEFBQAwRzELMAkGA1UEBhMCSEsx
-FjAUBgNVBAoTDUhvbmdrb25nIFBvc3QxIDAeBgNVBAMTF0hvbmdrb25nIFBvc3Qg
-Um9vdCBDQSAxMB4XDTAzMDUxNTA1MTMxNFoXDTIzMDUxNTA0NTIyOVowRzELMAkG
-A1UEBhMCSEsxFjAUBgNVBAoTDUhvbmdrb25nIFBvc3QxIDAeBgNVBAMTF0hvbmdr
-b25nIFBvc3QgUm9vdCBDQSAxMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKC
-AQEArP84tulmAknjorThkPlAj3n54r15/gK97iSSHSL22oVyaf7XPwnU3ZG1ApzQ
-jVrhVcNQhrkpJsLj2aDxaQMoIIBFIi1WpztUlVYiWR8o3x8gPW2iNr4joLFutbEn
-PzlTCeqrauh0ssJlXI6/fMN4hM2eFvz1Lk8gKgifd/PFHsSaUmYeSF7jEAaPIpjh
-ZY4bXSNmO7ilMlHIhqqhqZ5/dpTCpmy3QfDVyAY45tQM4vM7TG1QjMSDJ8EThFk9
-nnV0ttgCXjqQesBCNnLsak3c78QA3xMYV18meMjWCnl3v/evt3a5pQuEF10Q6m/h
-q5URX208o1xNg1vysxmKgIsLhwIDAQABoyYwJDASBgNVHRMBAf8ECDAGAQH/AgED
-MA4GA1UdDwEB/wQEAwIBxjANBgkqhkiG9w0BAQUFAAOCAQEADkbVPK7ih9legYsC
-mEEIjEy82tvuJxuC52pF7BaLT4Wg87JwvVqWuspube5Gi27nKi6Wsxkz67SfqLI3
-7piol7Yutmcn1KZJ/RyTZXaeQi/cImyaT/JaFTmxcdcrUehtHJjA2Sr0oYJ71clB
-oiMBdDhViw+5LmeiIAQ32pwL0xch4I+XeTRvhEgCIDMb5jREn5Fw9IBehEPCKdJs
-EhTkYY2sEJCehFC78JZvRZ+K88psT/oROhUVRsPNH4NbLUES7VBnQRM9IauUiqpO
-fMGx+6fWtScvl6tu4B3i0RwsH0Ti/L6RoZz71ilTc4afU9hDDl3WY4JxHYB0yvbi
-AmvZWg==
------END CERTIFICATE-----
-
 # 9a6ec012e1a7da9dbe34194d478ad7c0db1822fb071df12981496ed104384113
 Certificate:
     Data:
@@ -12087,193 +11883,6 @@ MjxNBiLOFeXdntiP2t7SxDnlF4HPOEfrf4htWRvfn0IUrn7PqLBmZdo3r5+qPeoo
 tt7VMVgWglvquxl1AnMaykgaIZOQCo6ThKd9OyMYkomgjaw=
 -----END CERTIFICATE-----
 
-# 873f4685fa7f563625252e6d36bcd7f16fc24951f264e47e1b954f4908cdca13
-Certificate:
-    Data:
-        Version: 3 (0x2)
-        Serial Number:
-            26:46:19:77:31:e1:4f:6f:28:36:de:39:51:86:e6:d4:97:88:22:c1
-        Signature Algorithm: ecdsa-with-SHA384
-        Issuer: C = TR, L = Ankara, O = E-Tugra EBG A.S., OU = E-Tugra Trust Center, CN = E-Tugra Global Root CA ECC v3
-        Validity
-            Not Before: Mar 18 09:46:58 2020 GMT
-            Not After : Mar 12 09:46:58 2045 GMT
-        Subject: C = TR, L = Ankara, O = E-Tugra EBG A.S., OU = E-Tugra Trust Center, CN = E-Tugra Global Root CA ECC v3
-        Subject Public Key Info:
-            Public Key Algorithm: id-ecPublicKey
-                Public-Key: (384 bit)
-                pub:
-                    04:8e:98:29:bf:c7:10:1e:27:db:ab:03:cc:28:2c:
-                    d8:5e:48:19:10:29:cc:cb:59:81:cc:8c:b8:92:17:
-                    89:83:2a:92:f6:c3:a4:1d:4c:62:d5:9f:d6:a0:46:
-                    dc:1c:bc:76:c1:e3:47:d0:5b:13:da:e7:a5:b3:66:
-                    48:e7:21:9a:4a:4f:86:0a:7d:6c:ea:4d:32:80:0a:
-                    b2:7a:09:9b:69:4b:98:81:e2:2e:ec:02:70:96:1f:
-                    fd:f5:46:ce:ca:dc:82
-                ASN1 OID: secp384r1
-                NIST CURVE: P-384
-        X509v3 extensions:
-            X509v3 Basic Constraints: critical
-                CA:TRUE
-            X509v3 Authority Key Identifier:
-                FF:82:31:72:3E:F9:C4:66:6C:AD:38:9E:D1:B0:51:88:A5:90:CC:F5
-            X509v3 Subject Key Identifier:
-                FF:82:31:72:3E:F9:C4:66:6C:AD:38:9E:D1:B0:51:88:A5:90:CC:F5
-            X509v3 Key Usage: critical
-                Certificate Sign, CRL Sign
-    Signature Algorithm: ecdsa-with-SHA384
-    Signature Value:
-        30:66:02:31:00:e6:05:58:69:61:e5:2d:ca:0d:cb:f1:19:08:
-        bd:d6:fd:51:92:1a:7e:63:54:04:90:91:9a:35:91:39:99:fa:
-        07:a9:66:93:ba:c8:68:d4:8a:3f:fa:ed:6e:16:02:27:b7:02:
-        31:00:dd:5a:17:2b:76:1d:65:42:96:a6:ac:5d:8a:79:56:d8:
-        8a:1b:df:9a:de:5f:c7:50:8f:b1:5b:71:0c:26:df:6a:40:00:
-        ec:33:91:21:71:be:68:e4:23:a4:d9:ad:a1:37
------BEGIN CERTIFICATE-----
-MIICpTCCAiqgAwIBAgIUJkYZdzHhT28oNt45UYbm1JeIIsEwCgYIKoZIzj0EAwMw
-gYAxCzAJBgNVBAYTAlRSMQ8wDQYDVQQHEwZBbmthcmExGTAXBgNVBAoTEEUtVHVn
-cmEgRUJHIEEuUy4xHTAbBgNVBAsTFEUtVHVncmEgVHJ1c3QgQ2VudGVyMSYwJAYD
-VQQDEx1FLVR1Z3JhIEdsb2JhbCBSb290IENBIEVDQyB2MzAeFw0yMDAzMTgwOTQ2
-NThaFw00NTAzMTIwOTQ2NThaMIGAMQswCQYDVQQGEwJUUjEPMA0GA1UEBxMGQW5r
-YXJhMRkwFwYDVQQKExBFLVR1Z3JhIEVCRyBBLlMuMR0wGwYDVQQLExRFLVR1Z3Jh
-IFRydXN0IENlbnRlcjEmMCQGA1UEAxMdRS1UdWdyYSBHbG9iYWwgUm9vdCBDQSBF
-Q0MgdjMwdjAQBgcqhkjOPQIBBgUrgQQAIgNiAASOmCm/xxAeJ9urA8woLNheSBkQ
-KczLWYHMjLiSF4mDKpL2w6QdTGLVn9agRtwcvHbB40fQWxPa56WzZkjnIZpKT4YK
-fWzqTTKACrJ6CZtpS5iB4i7sAnCWH/31Rs7K3IKjYzBhMA8GA1UdEwEB/wQFMAMB
-Af8wHwYDVR0jBBgwFoAU/4Ixcj75xGZsrTie0bBRiKWQzPUwHQYDVR0OBBYEFP+C
-MXI++cRmbK04ntGwUYilkMz1MA4GA1UdDwEB/wQEAwIBBjAKBggqhkjOPQQDAwNp
-ADBmAjEA5gVYaWHlLcoNy/EZCL3W/VGSGn5jVASQkZo1kTmZ+gepZpO6yGjUij/6
-7W4WAie3AjEA3VoXK3YdZUKWpqxdinlW2Iob35reX8dQj7FbcQwm32pAAOwzkSFx
-vmjkI6TZraE3
------END CERTIFICATE-----
-
-# ef66b0b10a3cdb9f2e3648c76bd2af18ead2bfe6f117655e28c4060da1a3f4c2
-Certificate:
-    Data:
-        Version: 3 (0x2)
-        Serial Number:
-            0d:4d:c5:cd:16:22:95:96:08:7e:b8:0b:7f:15:06:34:fb:79:10:34
-        Signature Algorithm: sha256WithRSAEncryption
-        Issuer: C = TR, L = Ankara, O = E-Tugra EBG A.S., OU = E-Tugra Trust Center, CN = E-Tugra Global Root CA RSA v3
-        Validity
-            Not Before: Mar 18 09:07:17 2020 GMT
-            Not After : Mar 12 09:07:17 2045 GMT
-        Subject: C = TR, L = Ankara, O = E-Tugra EBG A.S., OU = E-Tugra Trust Center, CN = E-Tugra Global Root CA RSA v3
-        Subject Public Key Info:
-            Public Key Algorithm: rsaEncryption
-                Public-Key: (4096 bit)
-                Modulus:
-                    00:a2:66:f0:89:b7:72:7b:ee:09:c9:63:d2:d3:43:
-                    dd:5e:c3:a6:84:38:4a:f1:8d:81:bb:14:bd:47:e8:
-                    40:17:f3:3d:c3:78:45:72:a6:2e:90:de:9a:3a:d4:
-                    20:71:ca:bc:9f:1d:4b:97:0a:c7:31:ba:3e:d7:fe:
-                    25:a9:2a:8e:36:f4:d1:2f:c7:b7:a9:5d:33:dc:30:
-                    70:f8:40:6c:4b:b2:a6:31:61:d1:34:3c:3d:31:7a:
-                    c7:af:c4:a7:a7:84:e1:97:a4:e8:4b:f6:17:7c:ee:
-                    3c:07:ed:e2:8a:57:dc:b6:fb:f8:43:25:50:ea:27:
-                    81:a8:86:bc:8f:52:4a:96:3a:60:1a:96:bb:fd:73:
-                    f4:85:fd:83:fd:7f:84:6d:34:6c:7f:6a:b7:4b:01:
-                    03:bf:ad:69:b7:d7:32:d9:f5:57:6a:e9:86:82:3e:
-                    a5:66:31:b3:16:3d:c2:f3:26:60:32:d3:52:1e:b0:
-                    6c:a4:37:3e:f4:f5:af:eb:e1:df:80:06:cf:2a:41:
-                    e7:66:09:e1:4b:97:e7:77:bd:21:6d:29:b6:67:c3:
-                    2d:7e:ed:d6:79:65:d1:cf:3a:b6:d1:b1:5e:56:61:
-                    50:7a:5a:ce:4e:50:31:80:03:98:47:e7:e4:18:7c:
-                    44:5a:c6:a4:b3:3b:c6:c6:c3:3a:f0:6c:c3:8b:c8:
-                    a4:91:05:f3:f5:d9:b6:aa:06:a1:b7:ab:e4:b1:ea:
-                    21:14:5c:83:a4:fc:ff:b6:50:d3:8c:12:26:99:76:
-                    70:e9:c0:0f:a6:74:fc:bb:d0:1b:78:ce:72:92:e2:
-                    28:9c:bc:e6:e9:09:d8:3a:d3:89:e6:be:2e:77:df:
-                    01:0a:6f:96:f6:e5:8d:3c:4d:52:76:1a:56:e1:73:
-                    7e:17:ac:3d:ad:6c:a3:52:12:18:70:e6:80:4e:33:
-                    f2:7e:26:32:ac:05:8d:38:a4:e6:76:3c:9f:10:69:
-                    0e:6d:9d:d2:c1:79:20:6b:5b:cf:33:8d:d1:94:76:
-                    35:e7:5d:55:c7:b7:ac:28:ab:46:cc:e7:3b:21:b5:
-                    0a:0a:e4:4a:59:dc:81:35:4b:44:95:12:0a:67:a5:
-                    a1:ff:5b:00:07:d2:c0:cc:f9:3f:fc:9f:33:f2:00:
-                    f8:8c:6c:87:9d:06:2d:f1:ef:e3:e6:06:fa:c5:66:
-                    13:5b:fc:50:07:9e:71:86:b2:da:6f:74:30:cf:93:
-                    53:e8:dc:22:d6:de:20:1f:61:8d:a3:2e:a3:78:32:
-                    90:6c:dc:ac:32:b5:05:e4:f5:3c:33:0d:d6:e0:87:
-                    77:17:4c:9d:b0:d8:09:a8:0d:57:f7:44:85:f0:c8:
-                    04:be:5c:5d:5a:e3:17:8e:54:63:69:7f:49:74:64:
-                    05:8c:a3
-                Exponent: 65537 (0x10001)
-        X509v3 extensions:
-            X509v3 Basic Constraints: critical
-                CA:TRUE
-            X509v3 Authority Key Identifier:
-                B2:B4:AE:E6:2D:F7:26:D5:AA:75:2D:76:4B:C0:1B:53:21:D0:48:EF
-            X509v3 Subject Key Identifier:
-                B2:B4:AE:E6:2D:F7:26:D5:AA:75:2D:76:4B:C0:1B:53:21:D0:48:EF
-            X509v3 Key Usage: critical
-                Certificate Sign, CRL Sign
-    Signature Algorithm: sha256WithRSAEncryption
-    Signature Value:
-        89:a8:72:7f:8c:eb:ce:2e:18:c4:10:80:2d:10:0c:ff:fb:14:
-        cd:04:e0:14:3c:4e:9a:fb:9f:29:bf:22:9e:57:b9:82:73:12:
-        63:26:b5:cc:90:e9:d2:2a:29:ee:9c:2d:cc:2c:99:be:45:27:
-        e4:b1:71:ed:e4:38:95:31:41:f2:7d:7a:63:78:df:ca:36:16:
-        2f:82:88:9f:bc:11:47:4f:76:4d:c8:2d:8e:eb:df:2d:7c:4e:
-        3b:da:ae:f6:e3:da:5d:14:a6:ae:e8:85:44:9d:06:6e:8e:fb:
-        ef:7a:4a:6a:2d:2b:28:18:fe:bf:90:2c:75:16:9f:0f:ea:96:
-        7d:05:ee:9b:13:a5:44:6c:f8:03:d0:dd:23:e1:fd:03:12:12:
-        08:f4:18:34:b3:e0:37:0b:77:11:01:48:bf:61:b4:b5:f8:19:
-        d9:cb:4d:ea:a3:8c:ef:fd:f0:06:b5:6d:92:f4:4a:61:50:84:
-        ed:ec:49:d3:e4:be:68:e6:2e:e3:31:0b:54:0b:1a:92:d6:82:
-        d8:b6:a2:65:3c:66:04:f9:55:da:6c:fb:db:b5:14:66:4d:94:
-        83:3b:cd:1e:a6:2b:b2:fe:77:40:86:ab:e7:df:0a:c9:fd:f6:
-        dd:87:56:18:d8:b0:2c:55:60:96:fa:08:7e:52:90:f5:4b:a6:
-        2e:87:7c:cb:20:db:06:3e:a0:5d:03:77:7d:a2:3c:13:1b:29:
-        a2:13:55:a0:3d:14:22:af:6f:b8:d0:9a:1b:72:dd:05:01:8d:
-        86:60:bf:a4:67:ee:b5:a5:0d:d1:7f:e6:1a:2b:62:66:c3:07:
-        ba:e7:a0:48:1c:38:c3:e9:45:fb:a7:7f:fc:ed:02:68:1a:ca:
-        77:12:77:a6:00:55:28:14:ec:d6:c7:12:a2:1b:65:42:e9:91:
-        e8:cb:3e:87:89:54:5d:d9:af:9d:97:9c:69:e7:0a:ff:0f:5a:
-        78:8b:63:2a:4c:7d:47:94:3f:de:4b:e9:53:d0:30:f1:c5:f6:
-        9e:49:df:3b:a0:91:a3:a3:fe:cd:58:cc:ea:df:af:6f:28:3b:
-        a0:69:9b:8f:ec:ac:ae:2b:54:9d:9b:04:b1:47:20:af:96:12:
-        3e:63:94:1d:04:e7:2e:bb:86:c7:0c:9a:88:bf:76:47:ef:f7:
-        b0:0b:97:66:d2:44:cf:60:52:07:e1:d5:2c:4a:3a:27:61:77:
-        ca:d7:8f:e7:87:0e:30:ff:0c:bb:04:e2:61:c3:a2:c8:97:61:
-        8e:b4:30:6a:3c:6d:c2:07:5f:4a:73:2f:3f:f9:16:8a:01:66:
-        ef:ba:91:ca:52:57:7b:ae:d4:e6:0f:dd:0b:7a:7f:8b:9e:26:
-        20:cf:3b:ef:81:71:83:59
------BEGIN CERTIFICATE-----
-MIIF8zCCA9ugAwIBAgIUDU3FzRYilZYIfrgLfxUGNPt5EDQwDQYJKoZIhvcNAQEL
-BQAwgYAxCzAJBgNVBAYTAlRSMQ8wDQYDVQQHEwZBbmthcmExGTAXBgNVBAoTEEUt
-VHVncmEgRUJHIEEuUy4xHTAbBgNVBAsTFEUtVHVncmEgVHJ1c3QgQ2VudGVyMSYw
-JAYDVQQDEx1FLVR1Z3JhIEdsb2JhbCBSb290IENBIFJTQSB2MzAeFw0yMDAzMTgw
-OTA3MTdaFw00NTAzMTIwOTA3MTdaMIGAMQswCQYDVQQGEwJUUjEPMA0GA1UEBxMG
-QW5rYXJhMRkwFwYDVQQKExBFLVR1Z3JhIEVCRyBBLlMuMR0wGwYDVQQLExRFLVR1
-Z3JhIFRydXN0IENlbnRlcjEmMCQGA1UEAxMdRS1UdWdyYSBHbG9iYWwgUm9vdCBD
-QSBSU0EgdjMwggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQCiZvCJt3J7
-7gnJY9LTQ91ew6aEOErxjYG7FL1H6EAX8z3DeEVypi6Q3po61CBxyryfHUuXCscx
-uj7X/iWpKo429NEvx7epXTPcMHD4QGxLsqYxYdE0PD0xesevxKenhOGXpOhL9hd8
-7jwH7eKKV9y2+/hDJVDqJ4GohryPUkqWOmAalrv9c/SF/YP9f4RtNGx/ardLAQO/
-rWm31zLZ9Vdq6YaCPqVmMbMWPcLzJmAy01IesGykNz709a/r4d+ABs8qQedmCeFL
-l+d3vSFtKbZnwy1+7dZ5ZdHPOrbRsV5WYVB6Ws5OUDGAA5hH5+QYfERaxqSzO8bG
-wzrwbMOLyKSRBfP12baqBqG3q+Sx6iEUXIOk/P+2UNOMEiaZdnDpwA+mdPy70Bt4
-znKS4iicvObpCdg604nmvi533wEKb5b25Y08TVJ2Glbhc34XrD2tbKNSEhhw5oBO
-M/J+JjKsBY04pOZ2PJ8QaQ5tndLBeSBrW88zjdGUdjXnXVXHt6woq0bM5zshtQoK
-5EpZ3IE1S0SVEgpnpaH/WwAH0sDM+T/8nzPyAPiMbIedBi3x7+PmBvrFZhNb/FAH
-nnGGstpvdDDPk1Po3CLW3iAfYY2jLqN4MpBs3KwytQXk9TwzDdbgh3cXTJ2w2Amo
-DVf3RIXwyAS+XF1a4xeOVGNpf0l0ZAWMowIDAQABo2MwYTAPBgNVHRMBAf8EBTAD
-AQH/MB8GA1UdIwQYMBaAFLK0ruYt9ybVqnUtdkvAG1Mh0EjvMB0GA1UdDgQWBBSy
-tK7mLfcm1ap1LXZLwBtTIdBI7zAOBgNVHQ8BAf8EBAMCAQYwDQYJKoZIhvcNAQEL
-BQADggIBAImocn+M684uGMQQgC0QDP/7FM0E4BQ8Tpr7nym/Ip5XuYJzEmMmtcyQ
-6dIqKe6cLcwsmb5FJ+Sxce3kOJUxQfJ9emN438o2Fi+CiJ+8EUdPdk3ILY7r3y18
-Tjvarvbj2l0Upq7ohUSdBm6O++96SmotKygY/r+QLHUWnw/qln0F7psTpURs+APQ
-3SPh/QMSEgj0GDSz4DcLdxEBSL9htLX4GdnLTeqjjO/98Aa1bZL0SmFQhO3sSdPk
-vmjmLuMxC1QLGpLWgti2omU8ZgT5Vdps+9u1FGZNlIM7zR6mK7L+d0CGq+ffCsn9
-9t2HVhjYsCxVYJb6CH5SkPVLpi6HfMsg2wY+oF0Dd32iPBMbKaITVaA9FCKvb7jQ
-mhty3QUBjYZgv6Rn7rWlDdF/5horYmbDB7rnoEgcOMPpRfunf/ztAmgayncSd6YA
-VSgU7NbHEqIbZULpkejLPoeJVF3Zr52XnGnnCv8PWniLYypMfUeUP95L6VPQMPHF
-9p5J3zugkaOj/s1YzOrfr28oO6Bpm4/srK4rVJ2bBLFHIK+WEj5jlB0E5y67hscM
-moi/dkfv97ALl2bSRM9gUgfh1SxKOidhd8rXj+eHDjD/DLsE4mHDosiXYY60MGo8
-bcIHX0pzLz/5FooBZu+6kcpSV3uu1OYP3Qt6f4ueJiDPO++BcYNZ
------END CERTIFICATE-----
-
 # e74fbda55bd564c473a36b441aa799c8a68e077440e8288b9fa1e50e4bbaca11
 Certificate:
     Data:
diff --git a/net/data/ssl/chrome_root_store/root_store.md b/net/data/ssl/chrome_root_store/root_store.md
index 3562b7179d5c6..915f68261858e 100644
--- a/net/data/ssl/chrome_root_store/root_store.md
+++ b/net/data/ssl/chrome_root_store/root_store.md
@@ -1,6 +1,6 @@
 <!-- mdformat off(generated) -->
 <!-- mdlint off(generated) -->
-Version: 10
+Version: 11
 
 SHA 256 Hash | Subject | NotBefore | NotAfter
 ---|---|---|---
@@ -37,7 +37,6 @@ e3b6a2db2ed7ce48842f7ac53241c7b71d54144bfb40c11f3f1d0b42f5eea12d | CN=Certigna,O
 7e37cb8b4c47090cab36551ba6f45db840680fba166a952db100717f43053fc2 | CN=DigiCert Assured ID Root G3,OU=www.digicert.com,O=DigiCert Inc,C=US | 2013-08-01 | 2038-01-15
 cb3ccbb76031e5e0138f8dd39a23f9de47ffc35e43c1144cea27d46a5ab1cb5f | CN=DigiCert Global Root G2,OU=www.digicert.com,O=DigiCert Inc,C=US | 2013-08-01 | 2038-01-15
 e23d4a036d7b70e9f595b1422079d2b91edfbb1fb651a0633eaa8a9dc5f80703 | CN=CA Disig Root R2,O=Disig a.s.,L=Bratislava,C=SK | 2012-07-19 | 2042-07-19
-b0bfd52bb0d7d9bd92bf5d4dc13da255c02c542f378365ea893911f55e55f23c | CN=E-Tugra Certification Authority,OU=E-Tugra Sertifikasyon Merkezi,O=E-Tuğra EBG Bilişim Teknolojileri ve Hizmetleri A.Ş.,L=Ankara,C=TR | 2013-03-05 | 2023-03-03
 125609aa301da0a249b97a8239cb6a34216f44dcac9f3954b14292f2e8c8608f | CN=emSign Root CA - C1,OU=emSign PKI,O=eMudhra Inc,C=US | 2018-02-18 | 2043-02-18
 40f6af0346a99aa1cd1d555a4e9cce62c7f9634603ee406615833dc8c8d00367 | CN=emSign Root CA - G1,OU=emSign PKI,O=eMudhra Technologies Limited,C=IN | 2018-02-18 | 2043-02-18
 86a1ecba089c4a8d3bbe2734c612ba341d813e043cf9e8a862cd5c57a36bbe6b | CN=emSign ECC Root CA - G3,OU=emSign PKI,O=eMudhra Technologies Limited,C=IN | 2018-02-18 | 2043-02-18
@@ -66,7 +65,6 @@ d947432abde7b7fa90fc2e6b59101b1280e0e1c7e4e40fa3c6887fff57a7f4cf | CN=GTS Root R
 b085d70b964f191a73e4af0d54ae7a0e07aafdaf9b71dd0862138ab7325a24a2 | CN=GlobalSign,OU=GlobalSign ECC Root CA - R4,O=GlobalSign | 2012-11-13 | 2038-01-19
 8d25cd97229dbf70356bda4eb3cc734031e24cf00fafcfd32dc76eb5841c7ea8 | CN=GTS Root R2,O=Google Trust Services LLC,C=US | 2016-06-22 | 2036-06-22
 5a2fc03f0c83b090bbfa40604b0988446c7636183df9846e17101a447fb8efd6 | CN=Hongkong Post Root CA 3,O=Hongkong Post,L=Hong Kong,ST=Hong Kong,C=HK | 2017-06-03 | 2042-06-03
-f9e67d336c51002ac054c632022d66dda2e7e3fff10ad061ed31d8bbb410cfb2 | CN=Hongkong Post Root CA 1,O=Hongkong Post,C=HK | 2003-05-15 | 2023-05-15
 9a6ec012e1a7da9dbe34194d478ad7c0db1822fb071df12981496ed104384113 | CN=ACCVRAIZ1,OU=PKIACCV,O=ACCV,C=ES | 2011-05-05 | 2030-12-31
 ebc5570c29018c4d67b1aa127baf12f703b4611ebc17b7dab5573894179b93fa | OU=AC RAIZ FNMT-RCM,O=FNMT-RCM,C=ES | 2008-10-29 | 2030-01-01
 46edc3689046d53a453fb3104ab80dcaec658b2660ea1629dd7e867990648716 | CN=TUBITAK Kamu SM SSL Kok Sertifikasi - Surum 1,OU=Kamu Sertifikasyon Merkezi - Kamu SM,O=Turkiye Bilimsel ve Teknolojik Arastirma Kurumu - TUBITAK,L=Gebze - Kocaeli,C=TR | 2013-11-25 | 2043-10-25
@@ -133,8 +131,6 @@ b4585f22e4ac756a4e8612a1361c5d9d031a93fd84febb778fa3068b0fc42dc2 | CN=Certainly
 77b82cd8644c4305f7acc5cb156b45675004033d51c60c6202a8e0c33467d3a0 | CN=Certainly Root R1,O=Certainly,C=US | 2021-04-01 | 2046-04-01
 554153b13d2cf9ddb753bfbe1a4e0ae08d0aa4187058fe60a2b862b2e4b87bcb | CN=AC RAIZ FNMT-RCM SERVIDORES SEGUROS,OU=Ceres,O=FNMT-RCM,C=ES,2.5.4.97=#130f56415445532d51323832363030344a | 2018-12-20 | 2043-12-20
 fb8fec759169b9106b1e511644c618c51304373f6c0643088d8beffd1b997599 | SERIALNUMBER=G63287510,CN=ANF Secure Server Root CA,OU=ANF CA Raiz,O=ANF Autoridad de Certificacion,C=ES | 2019-09-04 | 2039-08-30
-873f4685fa7f563625252e6d36bcd7f16fc24951f264e47e1b954f4908cdca13 | CN=E-Tugra Global Root CA ECC v3,OU=E-Tugra Trust Center,O=E-Tugra EBG A.S.,L=Ankara,C=TR | 2020-03-18 | 2045-03-12
-ef66b0b10a3cdb9f2e3648c76bd2af18ead2bfe6f117655e28c4060da1a3f4c2 | CN=E-Tugra Global Root CA RSA v3,OU=E-Tugra Trust Center,O=E-Tugra EBG A.S.,L=Ankara,C=TR | 2020-03-18 | 2045-03-12
 e74fbda55bd564c473a36b441aa799c8a68e077440e8288b9fa1e50e4bbaca11 | CN=Security Communication ECC RootCA1,O=SECOM Trust Systems CO.\,LTD.,C=JP | 2016-06-16 | 2038-01-18
 2e44102ab58cb85419451c8e19d9acf3662cafbc614b6a53960a30f7d0e2eb41 | CN=TunTrust Root CA,O=Agence Nationale de Certification Electronique,C=TN | 2019-04-26 | 2044-04-26
 30fbba2c32238e2a98547af97931e550428b9b3f1c8eeb6633dcfa86c5b27dd3 | CN=vTrus ECC Root CA,O=iTrusChina Co.\,Ltd.,C=CN | 2018-07-31 | 2043-07-31
diff --git a/net/data/ssl/chrome_root_store/root_store.textproto b/net/data/ssl/chrome_root_store/root_store.textproto
index dc7cae2af0dcb..9b5be80db9adc 100644
--- a/net/data/ssl/chrome_root_store/root_store.textproto
+++ b/net/data/ssl/chrome_root_store/root_store.textproto
@@ -8,7 +8,7 @@
 
 # Version # should always be incremented up whenever this (or any pem file that
 # it references) is changed.
-version_major: 10
+version_major: 11
 
 # CN=Actalis Authentication Root CA, O=Actalis S.p.A./03358520967, L=Milan, C=IT
 # https://ssltest-a.actalis.it:8443
@@ -214,13 +214,6 @@ trust_anchors {
   sha256_hex: "e23d4a036d7b70e9f595b1422079d2b91edfbb1fb651a0633eaa8a9dc5f80703"
 }
 
-# CN=E-Tugra Certification Authority, OU=E-Tugra Sertifikasyon Merkezi, O=E-Tuğra EBG Bilişim Teknolojileri ve Hizmetleri A.Ş., L=Ankara, C=TR
-# https://sslev.e-tugra.com.tr
-trust_anchors {
-  sha256_hex: "b0bfd52bb0d7d9bd92bf5d4dc13da255c02c542f378365ea893911f55e55f23c"
-  ev_policy_oids: "2.16.792.3.0.4.1.1.4"
-}
-
 # CN=emSign Root CA - C1, O=eMudhra Inc, OU=emSign PKI, C=US
 trust_anchors {
   sha256_hex: "125609aa301da0a249b97a8239cb6a34216f44dcac9f3954b14292f2e8c8608f"
@@ -395,11 +388,6 @@ trust_anchors {
   ev_policy_oids: "2.23.140.1.1"
 }
 
-# CN=Hongkong Post Root CA 1, O=Hongkong Post, C=HK
-trust_anchors {
-  sha256_hex: "f9e67d336c51002ac054c632022d66dda2e7e3fff10ad061ed31d8bbb410cfb2"
-}
-
 # C=ES, O=ACCV, OU=PKIACCV, CN=ACCVRAIZ1
 trust_anchors {
   sha256_hex: "9a6ec012e1a7da9dbe34194d478ad7c0db1822fb071df12981496ed104384113"
@@ -494,7 +482,7 @@ trust_anchors {
 # https://goodevssl.wisekey.com
 trust_anchors {
   sha256_hex: "6b9c08e86eb0f767cfad65cd98b62149e5494a67f5845e7bd1ed019f27b86bd6"
-  ev_policy_oids: "2.16.756.5.14.7.4.8"
+  ev_policy_oids: "2.23.140.1.1"
 }
 
 # CN=OISTE WISeKey Global Root GC CA, OU=OISTE Foundation Endorsed, O=WISeKey, C=CH
@@ -789,16 +777,6 @@ trust_anchors {
   sha256_hex: "fb8fec759169b9106b1e511644c618c51304373f6c0643088d8beffd1b997599"
 }
 
-# CN=E-Tugra Global Root CA ECC v3, OU=E-Tugra Trust Center, O=E-Tugra EBG A.S., L=Ankara, C=TR
-trust_anchors {
-  sha256_hex: "873f4685fa7f563625252e6d36bcd7f16fc24951f264e47e1b954f4908cdca13"
-}
-
-# CN=E-Tugra Global Root CA RSA v3, OU=E-Tugra Trust Center, O=E-Tugra EBG A.S., L=Ankara, C=TR
-trust_anchors {
-  sha256_hex: "ef66b0b10a3cdb9f2e3648c76bd2af18ead2bfe6f117655e28c4060da1a3f4c2"
-}
-
 # CN=Security Communication ECC RootCA1, O=SECOM Trust Systems CO.,LTD., C=JP
 trust_anchors {
   sha256_hex: "e74fbda55bd564c473a36b441aa799c8a68e077440e8288b9fa1e50e4bbaca11"
